packages/web/src/scss/components/Tooltip/README.md

```diff
@@ -107,20 +107,6 @@ automatically, including smart position updates to ensure Tooltip visibility,
 we recommend involving an external library designed specifically for this
 purpose.
 
-### Placement
```

Comment:
we don't need this?

Reply:
Please, expand the code and you will see that this is a duplication.